test/TEST-35-LOGIN/test.sh

```diff
@@ -7,4 +7,14 @@ TEST_DESCRIPTION="LOGIN"
 # shellcheck source=test/test-functions
 . "${TEST_BASE_DIR:?}/test-functions"
 
+test_append_files() {
+    if command -v expect >/dev/null && command -v tclsh >/dev/null ; then
```

Comment:
Please move this in `install_basic_tools()` in test-functions. The testing image is shared with other tests, hence these will not be included when multiple tests are executed, like our CIs do.